### PR TITLE
Simplified lists creation in three places

### PR DESCRIPTION
**Pull request by Stranger6667**
Trivial changes. Unnecessary appends are removed

### CODE REVIEW

File: django/contrib/admin/widgets.py [hunk 1]
```diff
@@ -45,12 +45,15 @@ def render(self, name, value, attrs=None, choices=()):
...
-        output = [super(FilteredSelectMultiple, self).render(name, value, attrs, choices)]
-        output.append('<script type="text/javascript">addEvent(window, "load", function(e) {')
-        # TODO: "id_" is hard-coded here. This should instead use the correct
-        # API to determine the ID dynamically.
-        output.append('SelectFilter.init("id_%s", "%s", %s); });</script>\n'
-            % (name, escapejs(self.verbose_name), int(self.is_stacked)))
+        output = [
+            super(FilteredSelectMultiple, self).render(name, value, attrs, choices),
+            '<script type="text/javascript">addEvent(window, "load", function(e) {',
+            # TODO: "id_" is hard-coded here. This should instead use the correct
```
Comment (timgraham) [inline django/contrib/admin/widgets.py]: I usually wrap comments at 79 chars.

File: django/db/models/sql/compiler.py [hunk 2]
```diff
@@ -1084,8 +1084,10 @@ def as_sql(self):
             return '', ()
         table = self.query.tables[0]
         qn = self.quote_name_unless_alias
-        result = ['UPDATE %s' % qn(table)]
-        result.append('SET')
+        result = [
+            'UPDATE %s' % qn(table),
+            'SET',
```
Comment (timgraham) [inline django/db/models/sql/compiler.py]: I think 'SET' can be included in the first string (at least the tests pass on SQLite with this change)?

**Comment by timgraham** [comment]
Please use past tense in commit messages "Simplified..."

**Comment by Stranger6667** [comment]
Done! Also I've moved `result` variable to another place, because there are possibilities (exceptions and return statement), when it will be not used at all.

**Comment by timgraham** [comment]
merged in 92eced62cdbc0d74567d3b67f8d18d5f4e5603d1, thanks!

**Pull request closed without merging**